packages/components/src/Frame/components/SkipToContent.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from '../../Link'
+
+export const SkipToContentContainer = styled(Link)`
+  left: 0;
+  top: 0;
+  position: absolute;
+  transform: translateY(calc(-100% - 16px));
+  z-index: 1;
+  padding: 16px;
+  filter: drop-shadow(0px 2px 4px rgba(34, 34, 34, 0.24));
```

Comment:
Should we create alternative theme shadow definitions using `filter` instead of `box-shadow` @jpye-finch ? The problem here is that `Text` uses `box-shadow` for it's internal styling, which makes the box-shadow it's using override the theme heavy shadow. If we had something like: `theme.shadows.heavyFilter` we could use it here too

Reply:
oh ok cool. :ets leave as is.